modules/test/playwright/helpers/ObjectApiHelper.ts

```diff
@@ -45,6 +46,21 @@ export class ObjectApiHelper {
 		);
 	}
 
+	async postObjectDefinitionRandomObjectEntries(
```

Comment:
I don't see the benefit of using this method over the direct post call to the API Helpers

Reply:
the data and the liferayConfig reference is handled on the helper side, which I think is cleaner than putting it into just one test file 🤔